gcp/modules/gcp-secret-mgmt/main.tf

```diff
@@ -22,6 +22,7 @@ module "gcp-secret-mgmt" {
   encryption_keys    = "${var.encryption_keys}"
   storage_location   = "${var.storage_location}"
   keyring_name       = "${var.keyring_name}"
+  apply_audit_config = false
```

Comment:
This change looked a little odd to me, so I wanted to make sure it's expected and not e.g. merge weirdness.

Reply:
It is very much expected, it is needed to disable audit config application in `secret-mgmt` module: https://github.com/gpii-ops/exekube/pull/32.